docs/src/routes/docs/[...3]modules/transaction-preview.md

```diff
@@ -85,12 +181,24 @@ export type TransactionPreviewAPI = {
    * and it will be patched to allow for transaction previewing
    */
   patchProvider: (provider: PatchedEIP1193Provider) => PatchedEIP1193Provider
+  
   /**
    * Pass this method a standard EIP1193 provider
    * (such as an injected wallet from window.ethereum)
    * and it will be patched to allow for transaction previewing
    */
   init: (initializationOptions: TransactionPreviewInitOptions) => void
+  
+  /**
+   * Pass this method a transaction that would be passed to a wallet provider
```

Comment:
This method accepts a transaction meant for a wallet provider (created using libraries like Ethers or Web3), simulates the transaction and generates a corresponding UI.

@Adamj1232 maybe better way to word it??

Reply:
Yep I like that a lot better, gracias!!

Reply:
This was super helpful and helped me refine the other two as well. TS notes FTW!